mev_inspect/schemas/liquidations.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+from pydantic import BaseModel
+from .classified_traces import ClassifiedTrace
+
+
+class Liquidation(BaseModel):
+    liquidated_usr: str
+    collateral_address: str
+    collateral_amount: int
+    collateral_source: str
```

Comment:
what is collateral source? is it the token used as collateral?

Reply:
flashloan, dydx, etc